dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -660,4 +660,15 @@ ssl.update.certificate.error.useraction=Ensure the location is accessible by the
 ssl.certificate.add.CWPKI0830I=CWPKI0830I: Certificate with the {0} SubjectDN from the {1} environment variable is being added to the {2} keystore.
 ssl.certificate.add.CWPKI0830I.explanation=A certificate was retrieved from an environment variable in the configuration and is being added to the keystore file.
 ssl.certificate.add.CWPKI0830I.useraction=No action is required.
-# -------------------------------------------------------------------------------------------------
\ No newline at end of file
+# -------------------------------------------------------------------------------------------------
+ssl.protocol.error.CWPKI0831E=CWPKI0831E: The {0} SSL/TLS protocol cannot be used.  Extended error is: {1} 
+ssl.protocol.error.CWPKI0831E.explanation:The protocol provided cannot be used.  An error occurs when trying to get a SSLContext instance with the protocol value. 
+ssl.protocol.error.CWPKI0831E.useraction:Ensure the configuration uses valid SSL/TLS protocol values for this JVM.
+# -------------------------------------------------------------------------------------------------
+ssl.protocol.error.CWPKI0832E=CWPKI0832E: The {0} SSL/TLS protocol value provided cannot be used when configuring a list of SSL protocols values.
+ssl.protocol.error.CWPKI0832E.explanation:The protocol value is not valid in a protocol list grouping.
```

Comment:
Reviewed messages:

ssl.protocol.error.CWPKI0832E=CWPKI0832E: The {0} SSL/TLS protocol value that is provided cannot be used for configuring a list of SSL protocols values.
ssl.protocol.error.CWPKI0832E.explanation: The protocol value is not valid in a protocol list grouping.
ssl.protocol.error.CWPKI0832E.useraction: Ensure that the list of SSL/TLS protocols includes only values that are appropriate for a protocol list.